src/pages/workspace/accounting/qbo/advanced/QuickbooksAdvancedPage.tsx

```diff
@@ -129,7 +129,7 @@ function QuickbooksAdvancedPage({policy}: WithPolicyProps) {
                                     <OfflineWithFeedback pendingAction={pendingFields?.reimbursementAccountID}>
                                         <MenuItemWithTopDescription
                                             shouldShowRightIcon
-                                            title="Croissant Co Payroll Account" // TODO: set to the current selected value
+                                            title={reimbursementAccountID}
```

Comment:
same deal with [this one](https://github.com/Expensify/App/pull/40718/files#r1578262161) 😄

Reply:
<img width="400" alt="Screenshot 2024-04-24 at 1 26 38 PM" src="https://github.com/Expensify/App/assets/98560306/4cc096bc-841f-4d1a-8133-72d061906019">